### PR TITLE
Bug 1565872 - Fix miscellaneous typos and spelling inconsistencies

### PR DESCRIPTION
**Pull request by kyoshino**
Fix typos in the code and use American English intensively. Mostly in the templates and comments but there are some symbols as well.

## Bugzilla link

[Bug 1565872 - Fix miscellaneous typos and spelling inconsistencies](https://bugzilla.mozilla.org/show_bug.cgi?id=1565872)

### CODE REVIEW

File: extensions/BMO/lib/Reports/Triage.pm [hunk 1]
```diff
@@ -366,7 +366,7 @@ sub owners {
           @{$input->{component}}
           );
         $selected = 1
-          if (!ref $input->{componet} && $input->{component} eq $component->name);
+          if (!ref $input->{component} && $input->{component} eq $component->name);
```
Comment (kyoshino) [inline extensions/BMO/lib/Reports/Triage.pm]: This code has been broken due to this typo.

File: template/en/default/mfa/duo/verify.html.tmpl [hunk 2]
```diff
@@ -57,7 +57,7 @@ $(function() {
 
 <p>
   <b>[% reason FILTER html %]</b> requires verification.<br>
-  [% UNLESS is_enrolment %]
+  [% UNLESS is_enrollment %]
```
Comment (kyoshino) [inline template/en/default/mfa/duo/verify.html.tmpl]: This has also been broken, because `is_enrollment` is defined at the top of the template.

Comment (kyoshino) [reply template/en/default/mfa/duo/verify.html.tmpl]: This mismatch is due to #232.

File: Bugzilla/App/Command/report_ping.pm [hunk 3]
```diff
@@ -130,7 +130,7 @@ __END__
 
 =head1 NAME
 
-Bugzilla::App::Command::report_ping - descriptionsend a report ping to a url';
+Bugzilla::App::Command::report_ping - descriptionsend a report ping to a URL';
```
Comment (dklawren) [inline Bugzilla/App/Command/report_ping.pm]: s/descriptionsend/Send/

**Pull request merged**